DataRepo/tests/templates/models/bst/test_td.py

```diff
@@ -47,7 +47,7 @@ def test_bst_column_field(self):
         }
         # Ignoring leading/trailing whitespace characters from the template code...
         html = self.render_td_template(context).strip()
-        self.assertIn('<td class="table-cell-nobr nobr">', html)
+        self.assertIn('<td class="table-cell-nobr ">', html)
```

Comment:
C; Noticed that "`nobr`" was removed here and below, and thought it probably effects nothings, was wondering whether your indeed to remove a space-prefixed  "` nobr`"? (like below).  Just checking...
```suggestion
        self.assertIn('<td class="table-cell-nobr">', html)
```

Reply:
The space is needed or else the test will fail.  It is the result of a template conditional.